lib/components/settings/widgets/settings_card.dart

```diff
@@ -91,23 +91,26 @@ class SettingsCard extends SettingsElementModel {
         onToggle = null,
         super(type: SettingsElementModelType.customTrailing, key: key);
 
-  SettingsCard.custom({Key? key, this.content})
+  const SettingsCard.custom({Key? key, this.content})
       : leading = null,
         onTap = null,
         onToggle = null,
         subtitle = null,
         title = null,
         trailing = null,
+        value = false,
         super(type: SettingsElementModelType.custom, key: key);
 
   @override
   _SettingsCardState createState() => _SettingsCardState();
 }
 
 class _SettingsCardState extends State<SettingsCard> {
+  late bool _value;
   //Build SettingsCard widget
   @override
   Widget build(BuildContext context) {
+    _value = widget.value;
```

Comment:
a big no no, move this to initState and didUpdateWidget, please avoid updating state variables directly into the build method

Reply:
still misses the didUpdateWidget declaration, without it will become a mess later on